inc/options/class-wpseo-option-titles.php

```diff
@@ -58,6 +58,7 @@ class WPSEO_Option_Titles extends WPSEO_Option {
 		'breadcrumbs-prefix'            => '', // Text field.
 		'breadcrumbs-searchprefix'      => '', // Text field.
 		'breadcrumbs-sep'               => '&raquo;', // Text field.
+		'breadcrumbs-nav-wrapper'       => false,
```

Comment:
I am not sure if an option is the right solution for this. 

If it is for accessibility, we might consider doing this by default and give the user the possibility to disable it by a hook. /cc @moorscode